src/objects/RelationshipController.java

```diff
@@ -23,10 +23,7 @@
 import tables.UserTable;
 import tables.VisitTable;
 
-import ui.ErrorView;
-import ui.LoginView;
-import ui.SearchView;
-import ui.VisitingUserView;
+import ui.*;
```

Comment:
I'm not sure why theres a ui.* here, I think IntelliJ maybe did it for me. I can change it to do specific imports instead